float32 pan_cmd
float32 speed
int8 speed_cmd